gravity-forms/gw-disable-submit-button-until-required-fields-are-filled-out.php

```diff
@@ -63,7 +71,8 @@ public function script() {
 					self.runCheck = function() {
 
 						var submitButton = $( '#gform_submit_button_' + self.formId );
-
+						// For using this snippet with GSPC, remove above line and replace with the line below.
+						// var submitButton = $( '.single_add_to_cart_button' );
```

Comment:
In GPPA, this is the logic we use for finding the submit button. Might help eliminate the need for customizing for GSPC:

```
$form
		.find('.gform_footer, .gform_page_footer, .gfield--type-submit')
		.find('input[type="submit"], input[type="button"]');
```

Reply:
Add to Cart isn't the gform_footer. Also, it's `button[type="submit"]. Tested and updated! Thanks.